app/src/main/java/org/mozilla/rocket/content/news/data/NewsRepositoryProvider.kt

```diff
@@ -1,24 +1,24 @@
 package org.mozilla.rocket.content.news.data
 
+import android.content.Context
+import org.mozilla.rocket.content.news.data.dailyhunt.DailyHuntNewsRemoteDataSource
+import org.mozilla.rocket.content.news.data.dailyhunt.DailyHuntProvider
 import org.mozilla.rocket.content.news.data.newspoint.NewsPointNewsRemoteDataSource
 import org.mozilla.rocket.content.news.data.rss.RssNewsRemoteDataSource
 
-class NewsRepositoryProvider {
+class NewsRepositoryProvider(private val appContext: Context) {
```

Comment:
Better to avoid make context as a dependency of the data layer classes. We could adjust them later.